app/cdap/components/WrangleHome/Components/WrangleCard/index.tsx

```diff
@@ -0,0 +1,145 @@
+import { Box, Card, Typography } from '@material-ui/core';
+import { fetchConnectors } from 'components/Connections/Create/reducer';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+import { Bigquery } from './iconStore/Bigquery';
+import { CloudSQLMySQL } from './iconStore/CloudSQLMySQL';
+import { CloudSQLPostGreSQL } from './iconStore/CloudSQLPostGreSQL';
+import { Database } from './iconStore/Database';
+import { GCS } from './iconStore/GCS';
+import { ImportDatasetIcon } from './iconStore/ImportDatasetIcon';
+import { Kafka } from './iconStore/Kafka';
+import { MySQL } from './iconStore/MySQL';
+import { Oracle } from './iconStore/Oracle';
+import { PostGRESSQL } from './iconStore/PostGRESQL';
+import { S3 } from './iconStore/S3';
+import { Spanner } from './iconStore/Spanner';
+import { SQLServer } from './iconStore/SQLServer';
+import { useStyles } from './styles';
+
+const WrangleCard = () => {
+  const [state, setState] = useState({
+    connectorTypes: [],
+  });
+  const getConnectorTypesNames = async () => {
+    let connectorTypes = await fetchConnectors();
+
+    connectorTypes = connectorTypes.map((connectorType) => {
+      if (connectorType.name === 'S3') {
```

Comment:
We don't want a hard-coded list of connectors, as users can add more via plugins.

Reply:
this will be covered as part of separate PR for Mui Icons after closure of this PR

Reply:
separate PR will be raised